### PR TITLE
[Magiclysm] Lot of different stuff

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None

#### Describe the solution

Sane some armor values of magic items, so they wont be on the same level as anti-tank armor
Add Gaia's Chosen spells - Terra Armor, that give [3*terra armor spell level] defense to all damage, and 0 lvl Dust Reborn spell, that create waypoint and ressurech character at this point afterwards
Replace old Frost Armor with new - almost no defence, but pretty nice damage
Sane the Heft effect to be not so crazy busted. Still good tho
Change the thunder pillar line attack to cone, so it can attack more enemies at once. Pretty busted, but will be fixed eventually with #62676
Move Soulfire EoC to enchantment folder, where all another EoC belongs
Tweaked the Earth Elemental attunement - now his spells do not require hands, can be casted faster, also the attunement got 3x  stamina and stamina regen boost, as his spells uses stamina intsead of a mana. Also earthquake now ignore walls and deal bash damage around, so be aware of your car nearby!
Added throwing knifes to knifeshot spell
Added NO_HANDS flag to knifeshot, because the fact you need to "throw" the knife with bare hands is stupid
Replaced the effect of holy armor and holy sword to new, conditional powered one
Added a cool integrated ethereal pocket for alchemist, that can store 10L or 20kg of mass for free

#### Testing

Everything works as intended, but im too lazy to describe each test

#### Additional context

(well, its Quake 35 lvl, you can't obtain such high level in the game currently, but it looks dope as for me)
image
image

### CODE REVIEW

**Comment by GuardianDll** [comment]
Probably need a string freeze label

File: data/mods/Magiclysm/enchantments/Gaias_Chosen.json [hunk 1]
```diff
@@ -27,5 +26,97 @@
...
+      }
+    ]
+  },
+  {
+    "id": "EOC_DUST_REBORN_2",
+    "type": "effect_on_condition",
+    "//": "This is two step EoC to make incorporeal effect trigger properly - without it you do not drop your items properly. Also remove all mutagen from your body, empty your fat (because its a backup body, contain not much of calories.)",
+    "condition": { "u_has_trait": "DUST_REBORN" },
+    "effect": [
+      { "u_teleport": { "u_val": "dust_reborn_teleport" }, "success_message": "debug teleport." },
```
Comment (GuardianDll) [inline data/mods/Magiclysm/enchantments/Gaias_Chosen.json:79]:
```suggestion
      { "u_teleport": { "u_val": "dust_reborn_teleport" } },
```

File: data/mods/Magiclysm/Spells/technomancer.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
   {
     "id": "holy_blade",
     "type": "SPELL",
-    "name": "Holy Blade",
+    "name": "Mage Blade",
     "description": "This blade of light will cut through any evil it makes contact with!",
```
Comment (Computer-Engineer-Man) [inline data/mods/Magiclysm/Spells/technomancer.json:55]: Is it supposed to still mention evil despite having the name Mage Blade?

Comment (Computer-Engineer-Man) [reply data/mods/Magiclysm/Spells/technomancer.json:55]: Perhaps something along the lines of "Blade of hardened mana" would be more apt.

File: data/mods/Magiclysm/Spells/technomancer.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
   {
     "id": "holy_blade_plus",
     "type": "SPELL",
-    "name": "Enhanced Holy Blade",
+    "name": "Enhanced Mage Blade",
     "description": "This blade of light will cut through any evil it makes contact with!  Now you know this spell like the back of your hand, and start to design your own version.",
```
Comment (Computer-Engineer-Man) [inline data/mods/Magiclysm/Spells/technomancer.json:80]: Same stuff here, probably changing light to say "hardened mana" or something.

File: data/mods/Magiclysm/Spells/technomancer.json [hunk 4]
```diff
@@ -100,7 +100,7 @@
   {
     "id": "spirit_armor",
     "type": "SPELL",
-    "name": "Spiritual Armor",
+    "name": "Mage Armor",
     "description": "Evil will not make it through your defenses if your faith is strong enough!",
```
Comment (Computer-Engineer-Man) [inline data/mods/Magiclysm/Spells/technomancer.json:104]: Don't see why evil would have relation to mage armor.

File: data/mods/Magiclysm/Spells/technomancer.json [hunk 5]
```diff
@@ -125,7 +125,7 @@
   {
     "id": "spirit_armor_plus",
     "type": "SPELL",
-    "name": "Enhanced Spiritual Armor",
+    "name": "Enhanced Mage Armor",
     "description": "Evil will not make it through your defenses if your faith is strong enough!  Now you know this spell like the back of your hand, and start to design your own version.",
```
Comment (Computer-Engineer-Man) [inline data/mods/Magiclysm/Spells/technomancer.json:129]: Don't see why evil or faith would have relation to mage armor again.

**Comment by sarseev** [comment]
A small question: how are we supposed to get the new spells, especially Terra Armor? Gaia's Chosen attunement doesn't automatically give it and, as far as I see, nothing does.

**Comment by GuardianDll** [comment]
fuck

**Pull request merged**